utils.js

```diff
@@ -17,3 +17,79 @@ export const SIZES = ['small', 'medium', 'large'];
 export const variantValidator = (variant) => {
   return COLOR_VARIANTS.includes(variant) !== -1;
 };
+
+//
+// Calendar Utils
+//
+/**
+ *
+ * @returns {Date} Today's date.
+ */
+export function getCurrentDate() {
```

Comment:
this seems like an unneeded abstraction - is it used much?

Reply:
agree. it's only used in Calendar.vue